deployments/common/image/common-scripts/env-conda

```diff
@@ -33,6 +33,7 @@ env_spec=/opt/environments/${env}/${env}.@(yml|explicit)
 
 env_frozen_yml=/opt/env-frozen/${env}/requirements.yml
 env_chilly_yml=/opt/env-chilly/${env}/requirements.yml
+hints=/opt/environments/${env}/buld-hints.conda
```

Comment:
Looks like there is a typo in the filename (build vs buld)

It doesn't matter since all the yml files are used but just letting you know.

Reply:
That's a great catch!  It would break the intended behavior of having a "conda level" build-hints file.   Except for the typing impaired.  :-)